server/server.js

```diff
@@ -18,8 +18,9 @@ if (process.env.LOG_LEVEL) {
 }
 
 // schema
-const buildSchema = require('./buildSchema')
+const buildSchema = require('data-sync-gql-core')
 const schemaListenerCreator = require('./lib/schemaListeners/schemaListenerCreator')
+const { makeExecutableSchema } = require('graphql-tools')
```

Comment:
It feels strange that API do not include this. I guess this is happening because of version conflicts in the package and it's clearly technical debt that will be resolved later?

Reply:
Yes exactly. It works without me needing to include this, but there are multiple errors mentioning unexpected results. I will try to find a solution, but yes - technical debt.

Reply:
This is resolved